web/src/components/tasks/business-formation/ReviewSection.tsx

```diff
@@ -0,0 +1,261 @@
+          <div className="maxw-mobile-lg margin-bottom-2">
+            <Content overrides={{ h3: headerLevelTwo }}>
+              {state.displayContent.reviewPageMembers.contentMd}
+            </Content>
+          </div>
+          <div className="margin-left-2">
+            <Button
+              style="tertiary"
+              onClick={() => {
+                setTab(2);
+                scrollToTop();
+              }}
+              underline
+              dataTestid="edit-members-section"
+            >
+              {BusinessFormationDefaults.editButtonText}
+            </Button>
+          </div>
+        </div>
+        {userData?.formationData.formationFormData.members.map((member, index) => (
+          <div className="display-block tablet:display-flex" key={`${member.name}-${index}`}>
+            <div className={`text-bold width-11rem ${index !== 0 ? "margin-top-1" : ""}`}>
+              <Content>{state.displayContent.reviewPageMemberName.contentMd}</Content>
+            </div>
+            <div className={index !== 0 ? "tablet:margin-top-1" : ""}>{member.name}</div>
+          </div>
+        ))}
+        <hr className="bg-base-lighter margin-y-205" />
+      </>
+    );
+  };
+
+  const getSignatures = () => (
+    <>
+      <div className="flex space-between">
+        <div className="maxw-mobile-lg margin-bottom-2">
+          <Content overrides={{ h3: headerLevelTwo }}>
+            {state.displayContent.reviewPageSignatures.contentMd}
+          </Content>
+        </div>
+        <div className="margin-left-2">
+          <Button
+            style="tertiary"
+            onClick={() => {
+              setTab(2);
+              scrollToTop();
+            }}
+            underline
+            dataTestid="edit-signature-section"
+          >
+            {BusinessFormationDefaults.editButtonText}
+          </Button>
+        </div>
+      </div>
+      <div className="display-block tablet:display-flex">
+        <div className="text-bold width-11rem">
+          <Content>{state.displayContent.reviewPageSignerName.contentMd}</Content>
+        </div>
+        <div>{state.formationFormData.signer}</div>
+      </div>
+      {userData?.formationData.formationFormData.additionalSigners.map((signer, index) => (
+        <div className="display-block tablet:display-flex" key={`${signer}-${index}`}>
+          <div className="text-bold width-11rem margin-top-1">
+            <Content>{state.displayContent.reviewPageSignerName.contentMd}</Content>
+          </div>
+          <div className="tablet:margin-top-1">{signer}</div>
+        </div>
+      ))}
+    </>
+  );
+
+  if (!userData || state.tab !== 3) return <></>;
+
+  return (
+    <>
+      <div data-testid="review-section">
+        <BusinessNameAndLegalStructure reviewPage />
+        {getBusinessSuffixAndStartDate()}
+        {getMainBusinessLocation()}
+        {getRegisteredAgent()}
+        {getMembers()}
+        {getSignatures()}
```

Comment:
since none of these functions are ever taking any arguments, would it be cleaner to just use variables intsead?

ie, `const businessSuffixAndStartDate = <>...</>` instead of the function

Reply:
updated